examples/x86_64.swiftinterface

```diff
@@ -0,0 +1,507 @@
+  case text
+  case html
+  case topicUpdated
+  case participantAdded
+  case participantRemoved
+  public var requestString: Swift.String {
+    get
+  }
+  public init(_ val: Swift.String)
+  public init(from decoder: Swift.Decoder) throws
+  public func encode(to encoder: Swift.Encoder) throws
+  public static func == (a: AzureCommunicationChat.ChatMessageType, b: AzureCommunicationChat.ChatMessageType) -> Swift.Bool
+}
+public struct ChatThreadProperties : Swift.Codable {
+  public let id: Swift.String
+  public let topic: Swift.String
+  public let createdOn: AzureCore.Iso8601Date
+  public let createdBy: AzureCommunicationCommon.CommunicationIdentifier
+  public let deletedOn: AzureCore.Iso8601Date?
+  public init(id: Swift.String, topic: Swift.String, createdOn: AzureCore.Iso8601Date, createdBy: AzureCommunicationCommon.CommunicationIdentifier, deletedOn: AzureCore.Iso8601Date? = nil)
+  public init(from decoder: Swift.Decoder) throws
+  public func encode(to encoder: Swift.Encoder) throws
+}
+public struct CreateChatThreadResult : Swift.Codable {
+  public let chatThread: AzureCommunicationChat.ChatThreadProperties?
+  public let invalidParticipants: [AzureCommunicationChat.ChatError]?
+  public init(chatThread: AzureCommunicationChat.ChatThreadProperties? = nil, invalidParticipants: [AzureCommunicationChat.ChatError]? = nil)
+  public func encode(to encoder: Swift.Encoder) throws
+  public init(from decoder: Swift.Decoder) throws
+}
+public struct SendChatMessageOptions : AzureCore.RequestOptions {
+  public let clientRequestId: Swift.String?
+  public let cancellationToken: AzureCore.CancellationToken?
+  public var dispatchQueue: Dispatch.DispatchQueue?
+  public var context: AzureCore.PipelineContext?
+  public init(clientRequestId: Swift.String? = nil, cancellationToken: AzureCore.CancellationToken? = nil, dispatchQueue: Dispatch.DispatchQueue? = nil, context: AzureCore.PipelineContext? = nil)
+}
+public struct SendChatMessageResult : Swift.Codable {
+  public let id: Swift.String
+  public init(id: Swift.String)
+  public init(from decoder: Swift.Decoder) throws
+  public func encode(to encoder: Swift.Encoder) throws
+}
+public struct CommunicationUserIdentifierModel : Swift.Codable {
+  public let id: Swift.String
+  public init(id: Swift.String)
+  public init(from decoder: Swift.Decoder) throws
+  public func encode(to encoder: Swift.Encoder) throws
+}
+public struct AddChatParticipantsOptions : AzureCore.RequestOptions {
+  public let clientRequestId: Swift.String?
+  public let cancellationToken: AzureCore.CancellationToken?
+  public var dispatchQueue: Dispatch.DispatchQueue?
+  public var context: AzureCore.PipelineContext?
+  public init(clientRequestId: Swift.String? = nil, cancellationToken: AzureCore.CancellationToken? = nil, dispatchQueue: Dispatch.DispatchQueue? = nil, context: AzureCore.PipelineContext? = nil)
+}
+public struct CommunicationIdentifierModel : Swift.Codable {
+  public let rawId: Swift.String?
```

Comment:
Why is this public? Is this the same value that `communicationUser.id` would have?

Reply:
I think in this case, if memory serves, `CommunicationIdentifierModel` can only have one of `communicationUser` or `phoneNumber` or `microsoftTeamsUser`.  
Also when we make a request using this object it is optional, but when we are decoding a response it is required.